src/modules/site-v2/templates/tools/heritability_calculator/result.html

```diff
@@ -151,7 +158,7 @@
             }
         }
 
-    var trait = "{{ trait }}"
+    var trait = "{{ hr.trait }}"
```

Comment:
prefer `let` and `const` when declaring new JS vars

Reply:
Fixed in 94bdca02fdd6f97a4d5ba2b37220d4be65abe2a8 (actually removed this variable)